styles/Home.module.css

```diff
@@ -1,36 +1,63 @@
 .container {
+	display: flex;
 	min-height: 100vh;
-	display: grid;
-	grid-template-columns: 50% auto;
-	grid-template-rows: 50% auto;
+	flex-direction: column;
+	color: var(--text-light);
+	background-color: var(--bg-light);
+	justify-content: center;
+}
+
+.header {
+	margin: 0 auto;
 }
 
 .main {
-	grid-column-start: 1;
-	grid-column-end: 1;
-	grid-row-start: 1;
-	grid-row-end: 2;
-	padding: 2.5rem 2.5rem;
-	flex: 1;
 	display: flex;
-	flex-direction: column;
-	justify-content: left;
-	align-items: flex-start;
+	flex: 0;
+	padding: 0 2.5rem 2.5rem 2.5rem;
+	margin: 0 auto;
 }
 
 .main a {
 	text-decoration: dotted;
 }
 
+.main > article {
+	flex: 1 0 auto;
+	max-width: 40vw;
+}
+
+.main > aside {
+	flex: 0 1 auto;
+}
+
+.extra {
+	display: flex;
+	flex-direction: row;
+	margin: 0 auto;
+	justify-content: center;
+}
+
+.extra > div {
+	flex-direction: column;
+	display: flex;
+	margin: 0 50px;
+	flex: 1 0;
+	height: 100%;
+	align-self: auto;
+}
+
 .title {
 	margin: 0;
+	padding: 0 2.5rem;
```

Comment:
title needs some padding for the top pls, the title is too close to the top https://hackropolis.is-ne.at/nA5HLM.png